### ISSUE TITLE
what is tensorrt build with float32 or float16 engine ?

### ISSUE THREAD

**Issue by SokPhanith** [dusty-nv/jetson-inference#1108]
sorry, I build and test tensorrt it's okay on jetson nano but I saw *.fp16 format on engine. It's meaning float16 engine right ? I want test with float32 engine but I don't know to set float32 for build.

**Comment by dusty-nv** [comment]
@SokPhanith yes, it means FP16.  If you are using C++ API, you can set the `precisionType` enum when you load the model:

https://github.com/dusty-nv/jetson-inference/blob/19ed62150b3e9499bad2ed6be1960dd38002bb7d/c/detectNet.h#L223

**Comment by SokPhanith** [comment]
@dusty-nv  Thank you it's working now.

**Comment by SehgalDivij** [comment]
@dusty-nv how can I set the same to FP32 for python applications?
